vizro-core/src/vizro/__init__.py

```diff
@@ -5,6 +5,6 @@
 
 __all__ = ["Vizro"]
 
-__version__ = "0.1.0"
+__version__ = "0.1.0dev0"
```

Comment:
Do we not go live with 0.1.0? or why do we change to the dev version?

Reply:
We will, this is just for development hygiene. Since we are not live, we should aim to be clear in that what people see on github is not v0.1.0, but the development version leading up to it.

I think it is even more important for the schema, if someone was to download and use it, that person would not know that then the schema for the actualy version 0.1.0 would be different